src/libraries/System.Private.CoreLib/src/System/AggregateException.cs

```diff
@@ -213,13 +213,14 @@ public override void GetObjectData(SerializationInfo info, StreamingContext cont
         }
 
         /// <summary>
-        /// Returns the <see cref="AggregateException"/> that is the root cause of this exception.
+        /// Returns the <see cref="Exception"/> that is the root cause of this exception.
+        /// This will either be the root exception, or the first <see cref="AggregateException"/>
+        /// that contains either multiple inner exceptions or no inner exceptions at all.
```

Comment:
These lines should go into remarks rather than into summary.

Reply:
@epsitec do you want to make that fix so we can merge?